server/controllers/notification.js

```diff
@@ -0,0 +1,95 @@
+const asyncHandler = require("express-async-handler");
+const Notification = require("../models/Notification");
+const User = require("../models/User");
+
+exports.createNotification = asyncHandler(async (req, res) => {
+  const date = new Date();
```

Comment:
Since we are using timestamps, is there any other reason why we want to store the date? :)

Reply:
deleted